### DISCUSSION TITLE
后台升级卡住了

### DISCUSSION THREAD

**Discussion by louiejordan** [lsky-org/lsky-pro#416] [问答]
卡在这20分钟了，而且还取消不了

![image](https://user-images.githubusercontent.com/17146050/167199625-d9905665-e0b7-4912-8836-9fd4e7e71894.png)

**Comment by louiejordan** [comment]
可以了，就是升级太慢了，记得1.x版本十几秒就升级好了

**Comment by 0xxb** [comment]
升级方式变的不一样了。
